packages/example-app-base/src/components/ComponentStatus/ComponentStatus.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import { IComponentStatusProps, TestCoverageStatus } from './ComponentStatus.Props';
+import './ComponentStatus.scss';
+
+export class ComponentStatus extends React.Component<IComponentStatusProps, {}> {
+
+  public static defaultProps: IComponentStatusProps = {
+    keyboardAccessibilitySupport: false,
+    markupSupport: false,
+    highContrastSupport: false,
+    rtlSupport: false,
+    testCoverage: TestCoverageStatus.none
+  };
+
+  public render(): JSX.Element {
+    const keyboardAccessibilitySubject = 'Keyboard Accessibility';
+    const markupSubject = 'Markup';
+    const highContrastSupportSubject = 'High Contrast';
+    const rtlSubject = 'Right to Left';
+    const testCoverageSubject = 'Test Coverage';
+
+    return (
+      <div className='ComponentStatus-div'>
+        <a
+          role='link'
+          aria-label={ 'Keyboard accessibility support. ' + this._badgeStatusString(this.props.keyboardAccessibilitySupport) }
+          href='#/components-status'
+          className='ComponentStatus-badge'
+        >
+          <img
+            src={ this._badgeURL(
+              this._colorForDualStatusBadge(this.props.keyboardAccessibilitySupport),
+              keyboardAccessibilitySubject,
+              this._badgeStatusString(this.props.keyboardAccessibilitySupport))
+            }
+          />
+        </a>
```

Comment:
The creation of each button is the same structure, consider pulling this into a helper function to reduce duplication

Reply:
also, you shouldn't need to mark this up with role="link" since it's actually using an anchor tag which has role="link" by default

Reply:
Yes, good point.